scripts/zones/Bastok_Markets/Zone.lua

```diff
@@ -17,32 +17,26 @@ end
 function onZoneIn(player, prevZone)
     local cs = -1
 
-    -- SOA 1-1 Optional CS
-    if
-        ENABLE_SOA and
-        player:getCurrentMission(SOA) == tpz.mission.id.soa.RUMORS_FROM_THE_WEST and
-        player:getCharVar("SOA_1_CS2") == 0
-    then
-        cs = 22
-    end
-
     -- FIRST LOGIN (START CS)
     if player:getPlaytime(false) == 0 then
         if NEW_CHARACTER_CUTSCENE == 1 then
-            --cs = 0
+            cs = 0
         end
         player:setPos(-280, -12, -91, 15)
         player:setHomePoint()
+    -- SOA 1-1 Optional CS
+    elseif
+        ENABLE_SOA and
```

Comment:
I coulda sword someone pr'd a fix to that check, since its 0 or 1 instead of nil/false/true